vuepress/docs/next/tutorials/solution/customer-portal.md

```diff
@@ -98,7 +98,7 @@ Go to the `CP Admin Config` page:
 3. From the `Action` drop-down menu on the right, go to `View Published Page`.
 
 ![Customer Portal Admin Config page](./images/cp-admin.png)
-Once the Ticketing System Connection is set up with Jira and the correct URL, default parameters such as product versions and ticket types will be displayed. Open each section with the down arrow to add and edit the fields as needed. 
+Once the Ticketing System Connection is set up with Jira and the correct URL, default parameters such as product versions and ticket types will be displayed. Open each section with the down arrow to add and edit the fields as needed. Note that the password for the Ticketing System Connection must be an [API token generated by Jira](https://id.atlassian.com/manage-profile/security/api-tokens) by the CP administrator. For more information, go to [Manange API tokens at Atlassian Support](https://support.atlassian.com/atlassian-account/docs/manage-api-tokens-for-your-atlassian-account/).
```

Comment:
"must be an API token generated by Jira by the CP administrator" is difficult to make sense of and ownership is muddy. something like this would be a simple fix: "Note that the password for the Ticketing System Connection must be an [API token generated by Jira](https://id.atlassian.com/manage-profile/security/api-tokens) **and requested** by the CP administrator."

Reply:
edited